src/EditorFeatures/Test/CommentSelection/CommentUncommentSelectionCommandHandlerTests.cs

```diff
@@ -390,18 +390,13 @@ class Foo
                 new TextChange(new TextSpan(20, 2), string.Empty),
                 new TextChange(new TextSpan(23, 2), string.Empty),
                 new TextChange(new TextSpan(38, 2), string.Empty),
-                new TextChange(new TextSpan(49, 2), string.Empty),
-                new TextChange(new TextSpan(52, 2), string.Empty),
                 new TextChange(new TextSpan(64, 2), string.Empty),
             };
 
             var expectedSelectedSpans = new[]
                 {
-                    Span.FromBounds(20, 21),
-                    Span.FromBounds(34, 34),
-                    Span.FromBounds(41, 42),
-                    Span.FromBounds(56, 56),
-                };
+                    Span.FromBounds(20, 21)
+                 };
 
             UncommentSelection(code, expectedChanges, expectedSelectedSpans, supportBlockComments: true);
```

Comment:
I would at least add a comment that says that the test is just measuring current behavior, and that there is no reason *not* to support maintaining box selection.